proposals/kubeflow-steering-committee-election-proposal.md

```diff
@@ -0,0 +1,125 @@
+The table below illustrates the cohorts and election schedules:
+
+<table>
+  <thead>
+    <tr>
+      <th>Timeline</th>
+      <th>2-member cohort</th>
+      <th>3-member cohort</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2023</td>
+      <td>Interim KSC member A<br>
+Interim KSC member B</td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>2024</td>
+      <td>New election</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>2025</td>
+      <td></td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>2026</td>
+      <td>New election</td>
+      <td></td>
+    </tr>
+    <tr>
+      <td>2027</td>
+      <td></td>
+      <td>New election</td>
+    </tr>
+    <tr>
+      <td>…</td>
+      <td></td>
+      <td></td>
+    </tr>
+  </tbody>
+</table>
+
+## Election Procedure
+
+### Timeline ###  
+Steering Committee elections are held annually. 4 weeks or more before the election, the Steering Committee will appoint Election Officer(s) (see below). 2 weeks or more before the election, the Election Officer(s) will issue a call for nominations, publish the list of eligible voters, and open the call for exceptions. One week before the election the call for nominations and exceptions will be closed. The election will be open for voting not less than two weeks and not more than four. The results of the election will be announced within one week of closing the election. New Steering Committee members will take office in December of each year on the date the results are announced.
+
+### Election Officer(s) ###  
+4 weeks or more before the election, the Steering Committee will appoint between one and three Election Officer(s) to administer the election. Elections Officers will be Kubeflow community members in good standing who are eligible to vote, are not running for Steering in that election, who are not currently part of the Steering Committee and can make a public promise of impartiality. They will be responsible for:
+
+-  Making all announcements associated with the election
+-  Preparing and distributing electronic ballots
+-  Judging exception requests
+-  Assisting candidates in preparing and sharing statements
+-  Tallying voting results according to the rules in this charter
+
+### Eligibility to Vote ###
+Anyone who has at least 50 contributions in the last 12 months is eligible to vote in the Steering election. Contributions are defined as opening PRs, reviewing and commenting on PRs, opening and commenting on issues, writing design docs, commenting on design docs, helping people on slack, participating in working groups, and other efforts that help advance the Kubeflow project.
```

Comment:
Contributions need not be just in dev. @andreyvelich

Reply:
I agree, we can explain here what kind of non dev contributions people can make to be eligible to vote.

Reply:
Maybe we can refer to CNCF devstats: https://kubeflow.devstats.cncf.io/d/8/dashboards?orgId=1&refresh=15m

Reply:
Borrowed this query from kNative election eligibility: https://kubeflow.devstats.cncf.io/d/9/developer-activity-counts-by-repository-group-table?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All

Reply:
I propose that those who want to vote will need to add their names to the voters.yaml file in electo, and that the addition / deletion will be approved by the election officer(s).   if the election officials needs justification, they can ask the voter to provide.

Reply:
and perhaps we can seed the list with the list that james has already.

Reply:
@jbottum the [elekto system has a process to apply for an exception to the voting eligibility built in](https://elekto.dev/docs/administration/running/#voter-exceptions), which I think is better than requiring PRs, because it removes the possibility of public shaming if the request is denied.